packages/gaming-ui/src/Components/Modules/DashboardModule.tsx

```diff
@@ -0,0 +1,259 @@
+        },
+        [breakpoints.up("sm")]: {
+          maxWidth: `calc(50% - ${constants.generalUnit * 2}px)`
+        },
+        [breakpoints.up("md")]: {
+          maxWidth: `calc(33% - ${constants.generalUnit * 2}px)`
+        },
+        [breakpoints.up("lg")]: {
+          maxWidth: `calc(25% - ${constants.generalUnit * 2}px)`
+        },
+        [breakpoints.up("xl")]: {
+          maxWidth: `calc(20% - ${constants.generalUnit * 2}px)`
+        }
+      }
+    },
+    modalRoot: {
+      zIndex: zIndex?.blocker,
+      [breakpoints.down("md")]: {}
+    },
+    modalInner: {
+      [breakpoints.down("md")]: {
+        bottom:
+        Number(constants?.mobileButtonHeight) + constants.generalUnit,
+        borderTopLeftRadius: `${constants.generalUnit * 1.5}px`,
+        borderTopRightRadius: `${constants.generalUnit * 1.5}px`,
+        maxWidth: `${breakpoints.width("md")}px !important`
+      }
+    },
+    modalHeading: {
+      textAlign: "center",
+      marginBottom: constants.generalUnit * 4
+    },
+    modalContent: {
+      display: "flex",
+      flexDirection: "column",
+      padding: constants.generalUnit * 4
+    },
+    secretContainer: {
+      display: "flex",
+      justifyContent: "space-between",
+      marginBottom: constants.generalUnit * 0.5
+    },
+    copyBox: {
+      display: "flex",
+      justifyContent: "space-between",
+      alignItems: "center",
+      cursor: "pointer",
+      color: palette.text.secondary
+    },
+    copyIcon: {
+      fontSize: "14px",
+      fill: constants.profile.icon,
+      [breakpoints.down("md")]: {
+        fontSize: "18px",
+        fill: palette.additional["gray"][9]
+      }
+    },
+    secret: {
+      maxWidth: "95%",
+      overflowWrap: "anywhere"
+    },
+    field: {
+      marginBottom: constants.generalUnit * 4
+    }
+  })
+)
+
+const DashboardModule = () => {
+  const classes = useStyles()
+  const { gamingApiClient } = useGamingApi()
+  const [keys, setKeys] = useState<AccessKey[]>([])
+  const [newKey, setNewKey] = useState<AccessKey | undefined>()
+  const [isNewKeyModalOpen, setIsNewKeyModalOpen] = useState(false)
+  const [copiedSecret, setCopiedSecret] = useState(false)
+  const debouncedCopiedSecret =
+    debounce(() => setCopiedSecret(false), 3000)
+
+  const copySecret = async () => {
+    if (newKey?.secret) {
+      try {
+        await navigator.clipboard.writeText(newKey.secret)
+        setCopiedSecret(true)
+        debouncedCopiedSecret()
+      } catch (err) {
+        console.error(err)
+      }
+    }
+  }
+
+  const fetchAccessKeys = useCallback(() => {
+    gamingApiClient.listAccessKeys()
+      .then(keys => setKeys(keys.filter(key => key.type === "gaming")))
+      .catch(console.error)
+  }, [gamingApiClient])
+
+  const createStorageAccessKey = useCallback(() => {
+    gamingApiClient.createAccessKey({ type: "gaming" })
+      .then(setNewKey)
+      .then(fetchAccessKeys)
+      .then(() => setIsNewKeyModalOpen(true))
```

Comment:
Any reason for 3 then?
```suggestion
      .then((key) => {
        setNewKey(key)
        fetchAccessKeys()
        setIsNewKeyModalOpen(true)
      })
```

Reply:
Nope, I just copied it from the Storage code as is